test/app-luatest/lua/gh_1700_abort_recording_on_fiber_switch.lua

```diff
@@ -0,0 +1,27 @@
+local hotloop = assert(arg[1], 'hotloop argument is missing')
+local trigger = assert(arg[2], 'trigger argument is missing')
+
+local ffi = require('ffi')
+local ffiyield = ffi.load('libyield')
+ffi.cdef('void yield(struct yield *state, int i)')
+
+-- Set the value to trigger <yield> call switch the running fiber.
+local yield = require('libyield')(trigger)
+
+-- Depending on trigger and hotloop values the following contexts
+-- are possible:
+-- * if trigger <= hotloop -> trace recording is aborted
+-- * if trigger >  hotloop -> trace is recorded but execution
+--   leads to panic
+jit.opt.start("3", string.format("hotloop=%d", hotloop))
```

Comment:
Minor: I suggest to use the following format to be consistent with the other code:
```lua
jit.opt.start(3, string.format('hotloop=%d', hotloop))
```
Side note: why do we set OPT level to default 3 here?

Feel free to ignore.

Reply:
I have no idea: the default OPT level is 3 IIRC, so I just removed it. Quotes are fixed.